src/common/POLFields/VendorFields/FieldRefNumberType.js

```diff
@@ -18,15 +19,35 @@ const REF_NUMBER_TYPE_OPTIONS = Object.keys(REF_NUMBER_TYPE).map((key) => ({
   value: REF_NUMBER_TYPE[key],
 }));
 
-const FieldRefNumberType = () => (
-  <FieldSelect
-    dataOptions={REF_NUMBER_TYPE_OPTIONS}
-    fullWidth
-    label={<FormattedMessage id="ui-orders.vendor.refNumberType" />}
-    name="vendorDetail.refNumberType"
-    validate={[requiredRefNumber]}
-  />
-);
+const FieldRefNumberType = ({ dispatch, change, required }) => {
+  const onChange = (_, value) => {
+    dispatch(change('vendorDetail.refNumberType', value));
```

Comment:
I still don't get why we need this `onChange` - your field is `vendorDetail.refNumberType` already.

Reply:
the same question

Reply:
Fixed